### PR TITLE
fix(cli): add cli core aliases, and two minor fixes

### PR DESCRIPTION
**Pull request by UnleashedMind**
some minor fixes: 
1. add back cli core aliases: "h", "ls", "serve"

2. relax input params requirements to allow plain string
instead of just Json object. 

3. fix a bug when awscloudformation configure is run directly instead of invoked by the core confgiure command

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-cli/src/lib/input-params-manager.js [hunk 1]
```diff
@@ -26,7 +26,12 @@ function normalizeKey(key) {
 }
 
 function normalizeValue(key, value) {
-  const normalizedValue = JSON.parse(value);
+  let normalizedValue = value;
+  try {
+    normalizedValue = JSON.parse(value);
+  } catch (e) {
+    // do nothing
```
Comment (kaustavghosh06) [inline packages/amplify-cli/src/lib/input-params-manager.js]: This change seems unrelated to the PR. What's this for?

File: packages/amplify-provider-awscloudformation/lib/configuration-manager.js [hunk 2]
```diff
@@ -30,6 +30,7 @@ async function init(context) {
 }
 
 async function configure(context) {
+  context.exeInfo = context.exeInfo || context.amplify.getProjectDetails();
```
Comment (kaustavghosh06) [inline packages/amplify-provider-awscloudformation/lib/configuration-manager.js]: This change seems unrelated to the PR. What's this for?

File: packages/amplify-cli/src/lib/input-params-manager.js (hunk 1 above)
Comment (UnleashedMind) [reply packages/amplify-cli/src/lib/input-params-manager.js]: It's in the description of the PR. 
It's too restrictive to only allow Json string as parameters. 
This update allows plain string as parameter, which is used for things such as sample's '--app' option.

File: packages/amplify-provider-awscloudformation/lib/configuration-manager.js (hunk 2 above)
Comment (UnleashedMind) [reply packages/amplify-provider-awscloudformation/lib/configuration-manager.js]: The provider plugin command: `amplify awscloudformation configure` is broken for a while, but I guess it's not directly used by the users, but invoked through "amplify configure project", which works,  so it's not  discovered. This fixes it the direct invocation of the configuration of the provider plugin.

**Comment by UnleashedMind** [comment]
This PR contains three separate updates because they are all small fixes. But it can also be separated into 3 individual PR's depending on the team's opinions.

File: packages/amplify-cli/src/lib/input-params-manager.js [hunk 3]
```diff
@@ -26,10 +26,12 @@ function normalizeKey(key) {
 }
 
 function normalizeValue(key, value) {
-  if (key === 'app') {
```
Comment (kaustavghosh06) [inline packages/amplify-cli/src/lib/input-params-manager.js]: Why are we removing this? This is a part of the latest release for the sample apps I believe. @nikhname could you please review this?

File: packages/amplify-cli/src/lib/input-params-manager.js (hunk 1 above)
Comment (nikhname) [reply packages/amplify-cli/src/lib/input-params-manager.js]: I'm not sure this is the best way to allow a plain string parameter...do we want to return the value if an invalid JSON is passed?

File: packages/amplify-cli/src/lib/input-params-manager.js (hunk 3 above)
Comment (nikhname) [reply packages/amplify-cli/src/lib/input-params-manager.js]: Yes, I don't think this should be removed. The workaround below is a general way to do this but it ignores errors in JSON.parse

File: packages/amplify-cli/src/lib/input-params-manager.js (hunk 1 above)
Comment (UnleashedMind) [reply packages/amplify-cli/src/lib/input-params-manager.js]: If the value can not be parsed as JSON, then it's passed as plain string.
